This patch was sent by Vyacheslav Perestoronin <perestoronin@gmail.com>

Declared purpose is to make VTK to build with CUDA-11.5 release.

--- a/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/internal/brigand.hpp
+++ b/ThirdParty/vtkm/vtkvtkm/vtk-m/vtkm/internal/brigand.hpp
@@ -1057,18 +1057,6 @@
         {
             using type = S<L, Ls...>;
         };
-        template <typename L1, typename L2, typename... Ls>
-        struct find<true, false, L1, L2, Ls...> : find<true, F<Ts..., L2>::value, L2, Ls...>
-        {
-        };
-        template <typename L0, typename L1, typename L2, typename L3, typename L4, typename L5,
-                  typename L6, typename L7, typename L8,
-                  typename... Ls>
-        struct find<false, false, L0, L1, L2, L3, L4, L5, L6, L7, L8, Ls...>
-            : find<true, F<Ts..., L8>::value, L8, Ls...>
-        {
-        };
-
 #ifndef BRIGAND_COMP_CUDA
         template <typename L1, typename L2, typename L3, typename L4, typename L5, typename L6,
                   typename L7, typename L8, typename L9, typename L10, typename L11, typename L12,
